assets/stylesheets/components/_skiplinks.scss

```diff
@@ -0,0 +1,22 @@
+.skiplinks {
+  background-color: tint($blue, 65%);
+}
+
+  .skiplinks--inner {
+    @extend %content-block;
+  }
+
+  .skiplinks--link {
+    color: $white;
+    display: inline-block;
+    left: -999em;
+    padding: ($baseline-grid-unit * 2);
+    position: absolute;
+
+    &:focus {
+      background-color: tint($blue, 95%);
+      border: 2px solid $yellow;
```

Comment:
Can we keep the focus outline/border colour consistent. We're using 3px for outline but only 2px here